main.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import glob
+import time
+
+from archive_converter import ArchiveMessageIndex
+from archive_updater import fill_message_directory
+from gerrit import upload_all_comments, get_gerrit_rest_api, Gerrit, find_and_label_all_revision_ids
+from git import GerritGit
+from message_dao import MessageDao
+from patch_parser import map_comments_to_gerrit, parse_comments, Patch, Patchset
+from setup_gmail import Message
+from typing import List, Set, Tuple
+
+GIT_PATH = '../linux-kselftest/git/0.git'
+FILE_DIR = 'index_files'
```

Comment:
fyi, the code currently errors out if this dir doesn't exist beforehand.

Might be useful to include a step somewhere to create it if it doesn't exist.
```python
os.makedirs(FILE_DIR, exist_ok=True)
```